xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
+## 1.0 Introduction
 
-<a name="roleofxapi"/>
-## 1.0 Summary of the Experience API
+The Experience API (xAPI) is a technical specification that aims to facilitate the documentation and communication of learning experiences.  It does so by setting out a protocol that specifies how learning experiences should be described and an Application Programming Interface that sets out how information about these experiences should be stored and exchanged electronically.  
 
-The Experience API (xAPI) is an Application Programming Interface (API) specification that 
-facilitates communication about experiences between a Learning Record Store and a client, 
-typically an Activity Provider. This specification was developed with learning experiences in mind, 
-but many other kinds of experiences could be tracked using xAPI. 
+The xAPI is part of the Advanced Distributed Learning (ADL) initiative, established in 1997 to standardize and modernize training and education management and delivery in the US. Among the many outcomes of this initiative was SCORM - the Shareable Content Object Reference Model. SCORM has 
```

Comment:
I would expand on US to either US DoD or US Government.  That was the mission, but luckily the work went so much further!